### ISSUE TITLE
CCR-CLIP pretraining

### ISSUE THREAD

**Issue by gyao15** [FudanVI/FudanOCR#84]
It seems that the image contrastive loss is not used in the provided codes and the number of pretraining epochs is not clear in both paper and codes (set to 10000 in the codes but obviously it is impossible to spend so much time on pretraining)

**Comment by hyangyu** [comment]
Thank you for your attention. The contrastive loss function between images is missing since the code version is not up-to-date. We will update the image contrastive loss. The number of epochs for training is obviously not 10000. Actually, we usually stop pre-training when the accuracy of the test set no longer increases.

**Comment by gyao15** [comment]
Thanks!
